recipes/sundials/all/conanfile.py

```diff
@@ -0,0 +1,85 @@
+from conans import CMake, ConanFile, tools
+import os
+
+
+class SundialsConan(ConanFile):
+    name = "sundials"
+    license = "BSD 3-Clause"
+    description = ("SUNDIALS is a family of software packages implemented"
+                   " with the goal of providing robust time integrators "
+                   "and nonlinear solvers that can easily be incorporated"
+                   "into existing simulation codes.")
+    topics = ("sundials", "integrators", "ode", "non-linear solvers")
+    homepage = "https://github.com/LLNL/sundials"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False],
+               "fPIC": [True, False],
+               "build_arkode": [True, False],
+               "build_cvode": [True, False],
+               "build_cvodes": [True, False],
+               "build_ida": [True, False],
+               "build_idas": [True, False],
+               "build_kinsol": [True, False]}
+    default_options = {"shared": False,
+                       "fPIC": False,
+                       "build_arkode": True,
+                       "build_cvode": True,
+                       "build_cvodes": True,
+                       "build_ida": True,
+                       "build_idas": True,
+                       "build_kinsol": True}
+    exports_sources = "CMakeLists.txt", "patches/**"
+    generators = "cmake"
+    short_paths = True
```

Comment:
```suggestion
```

Reply:
It doesn't compile in windows without this, due to very long names.

Reply:
fair enough